### ISSUE TITLE
Setting query-log-enabled = false does not disable query logging

### ISSUE THREAD

**Issue by jwheeler-gs** [influxdata/influxdb#7238]
### Bug report

I can't seem to find any way to disable query logging in influx. The configuration option for "query-log-enabled" in the documentation is described as:
"The query log enabled setting toggles the logging of parsed queries before execution. Very useful for troubleshooting, but will log any sensitive data contained within a query."

I would think that this option would disable the log statements that begin with [query] and output every query that's being run against the database.

__System info:__ Influx v0.13, docker influxdb or influxdb:alpine

__Steps to reproduce:__

Two ways to reproduce. Disabling query logging does not work by environment variable or by configuration file.
1. docker run --name influxdb -v /my/influx/path -p 8083:8083 -p 8086:8086 -e INFLUXDB_REPORTING_DISABLED=true -e INFLUXDB_DATA_QUERY_LOG_ENABLED=false -e INFLUXDB_HTTP_LOG_ENABLED=false -e INFLUXDB_CONTINUOUS_QUERIES_LOG_ENABLED=false influxdb:alpine 
2. See queries still being logged

Not working via config file:
1. docker run --name influxdb -v /my/influx/path -v $PWD/influxdb.conf:/etc/influxdb/influxdb.conf:ro -p 8083:8083 -p 8086:8086 influxdb:alpine
2. See queries still being logged

(relevant section from the config file)
```
[data]
  dir = "/var/lib/influxdb/data"
  engine = "tsm1"
  wal-dir = "/var/lib/influxdb/wal"
  wal-logging-enabled = false
  query-log-enabled = false
  cache-max-memory-size = 524288000
  cache-snapshot-memory-size = 26214400
  cache-snapshot-write-cold-duration = "1h0m0s"
  compact-full-write-cold-duration = "24h0m0s"
  max-points-per-block = 0
  data-logging-enabled = false
```

__Expected behavior:__ No query entries in the logs such as:
[query] 2016/08/29 20:52:59 SELECT mean(myfield) FROM "Example"."1h".devices WHERE (device = 'abc') AND (time >= now() - 5m) GROUP BY device, time(1s) fill(previous)

__Actual behavior:__ The query log messages are still being output by influx

Are there any known workarounds for this problem? Is this a documentation bug?

**Comment by jsternberg** [comment]
Have you tried this with the rc1? I can check again, but I think this might be fixed already.

**Comment by jwheeler-gs** [comment]
Just confirmed this is fixed in RC1. Is RC1 as stable to use in production as 0.13 is at this point?

**Comment by jsternberg** [comment]
I would wait until RC2 which should be hitting either today or tomorrow. It's more stable in general, but there's a decently nasty bug we found in RC1 that isn't in 0.13. beta3 should be stable enough if you can't wait for RC2 though.

**Comment by jsternberg** [comment]
Closing this since it is already resolved. Thanks for the bug report!

**Comment by manishg4** [comment]
Is there a work around to get this resolved in Influx v0.13, without needing to move to the later version?

**Comment by jsternberg** [comment]
You can find the change that fixed this, apply it to the v0.13 tag (if it applies correctly), then compile it. We are not going to backport this fix and release anything for v0.13 at this point.
